mbedtls/src/pk/mod.rs

```diff
@@ -313,70 +264,13 @@ impl Pk {
         let mut ret = Self::init();
         unsafe {
             pk_setup(&mut ret.inner, pk_info_from_type(Type::Eckey.into())).into_result()?;
-            let ctx = ret.inner.pk_ctx as *mut ecp_keypair;
-            (*ctx).grp = curve.into_inner();
-            (*ctx).Q = public_point.into_inner();
-        }
-        Ok(ret)
-    }
-
-    pub fn public_custom_algo(algo_id: &[u64], pk: &[u8]) -> Result<Pk> {
```

Comment:
Same, I believe we need this.

Reply:
Mark as resolved since:
Custom Pk implementation is now not include in this rust mbedtls wrapper library